openshift/README.md

```diff
@@ -0,0 +1,75 @@
+# Build and deploy on OpenShift Container Platform
+
+The loopback4-example-shopping/openshift contains Dockerfiles for building
+container images suitable for running on OpenShift Container Platform and yaml
+files for creating build configurations for building the images. They have been
+tested on OCP 4.6.9.
+
+## First build and deploy
+
+First run these commands in loopback4-example-shopping/openshift to create a new
+project called shoppy and to install the application and databases in the new
+project.
+
+```
+oc new-project shoppy
+oc new-app mongo --name shopping-app-mongodb
```

Comment:
is this password protected?

Reply:
Similarly, for Redis how does the password configuration work?

Reply:
There are no password protections for the databases.
The datasources expects that the databases are not password protected.
This is not something that is OpenShift specific, this is the current state of the application.
To start using passwords you have to change mango.datasource.ts and redis.datasource.ts.

The "--name shopping-app-mongodb" is for using the same solution as for Kubernetes example.